aws_lambda_builders/workflows/java_maven/DESIGN.md

```diff
@@ -71,21 +71,17 @@ configured to use a different one than can be found on the PATH.
 
 #### Step 3: Build and package
 
-We leverage Maven to do all the heavy lifting for executing the`mvn package` which
+We leverage Maven to do all the heavy lifting for executing the`mvn clean install` which
 will resolve and download the dependencies and build the project. Built java classes 
 will be located in `target/classes`. Then we use `mvn dependency:copy-dependenceis` to copy
 the dependencies and the dependencies will be located in `target/dependency` under the 
 source directory.
 
 ```bash
-MODULE_NAME=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.artifactId}' exec:exec --non-recursive)
-mvn clean package -pl :MODULE_NAME -am
-mvn dependency:copy-dependencies -DincludeScope=compile -pl :MODULE_NAME
+mvn clean install
```

Comment:
Just checking, this works on windows too right?

Reply:
Ya. I patched the windows machine that was failing before this change and was able to build locally on the windows machine with Maven.